package.json

```diff
@@ -34,9 +34,9 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
-    "@edx/frontend-component-footer": "11.2.0",
-    "@edx/frontend-component-header": "3.2.0",
-    "@edx/frontend-platform": "2.6.1",
+    "@edx/frontend-component-footer": "11.6.3",
+    "@edx/frontend-component-header": "3.6.4",
+    "@edx/frontend-platform": "3.4.1",
```

Comment:
The platform is now on 4.2.0, which is needed for the atlas PR: #502

Reply:
This is intentional since the owning squad suggested we bump these packages incrementally. Once merged, it will be followed by node v18 upgrade which will also include platform upgrade to v4.2.0

Reply:
Thanks Bilal. Looks good to me.